src/SFA.DAS.QnA.Api.Views/SFA.DAS.QnA.Api.Views.csproj

```diff
@@ -17,4 +17,8 @@
             <CopyToOutputDirectory>Always</CopyToOutputDirectory>
         </Content>
     </ItemGroup>
+  <ItemGroup Condition=" $(TargetFramework) == 'netstandard2.0'">
```

Comment:
I removed those packages above and googled the error message - adding the package below and restarting VS removed the error message for me which would be much better if it works
```
<ItemGroup Condition=" $(TargetFramework) == 'netstandard2.0'">
      <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
</ItemGroup>
```

Reply:
Change has been made